.github/meta.yaml

```diff
@@ -37,7 +37,7 @@ outputs:
         - requirements-parser >=0.2.0
         - shap >=0.40.0
         - texttable >=1.6.2
-        - woodwork >=0.15.0
```

Comment:
0.16.2 not available for download yet which is why the build conda pkg is failing. Should be fixed later though